generators/app/templates/src/app/shell/__ionic-tabs.shell.component.ts

```diff
@@ -35,12 +31,6 @@ export class ShellComponent {
     }
     if (route && route.component === ShellComponent && route.firstChild) {
       route = route.firstChild;
-      // Fixed the bug#19420 : route.component is undefined if module is lazy
-      // See: https://github.com/angular/angular/issues/19420
-      while (route.firstChild) {
```

Comment:
are you sure this can be removed if one the tab component is lazy loaded?

Reply:
yup :) I tested it, and in fact, it now doesn't work WITHOUT removing this.

Also, I think this was here because the ionic 3 version was using the component to match the tab instead of the URL (since lazy-loading didn't work anyway).  Now that we're using the tab we don't need (or want) the component ;)